third_party/gcompat/Makefile

```diff
@@ -0,0 +1,13 @@
+URL=https://github.com/AdelieLinux/gcompat.git
+
+TAG=1.0.0
+
+all: src
+
+clean: 
+	rm -rf gcompat
+
+src:
```

Comment:
This fails when I type make the second time. I renamed src to gcompat and pushed (this prevents it from trying to clone again if you type make the second time). 

I also added the **distclean** target, which is required if you type "make distclean" in the directory above.

Reply:
Thank you Mike.